packages/client/src/store/slices/forumSlice/thunks.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  CommentInfo,
+  CreateCommentData,
+  CreateTopicData,
+  TopicInfo,
+} from '@/api/Forum/types'
+import { IExtraArgument } from '@/store'
+import { createAsyncThunk } from '@reduxjs/toolkit'
+
+export const getAllTopics = createAsyncThunk(
+  'forum/getAllTopics',
+  async (_, thunkApi) => {
+    try {
+      const forumApi = (thunkApi.extra as IExtraArgument).forumService
+      const topicsList = await forumApi.getAllTopics()
+      return topicsList
+    } catch (error) {
+      return thunkApi.rejectWithValue(false)
+    }
+  }
+)
+
+export const addNewTopic = createAsyncThunk<TopicInfo, CreateTopicData>(
+  'forum/addNewTopic',
+  async (CreateTopicData, thunkApi) => {
```

Comment:
Переменные обычно с маленькой буквы
```suggestion
  async (createTopicData, thunkApi) => {
```